datanode/gateway/graphql/schema.graphql

```diff
@@ -3864,6 +4026,9 @@ union ProposalChange =
   | NewTransfer
   | CancelTransfer
   | UpdateMarketState
```

Comment:
**question (non-blocking)**:  is a new perpetual market just a `NewMarket`, or like spot is it getting a `NewSpotMarket`? 

It seems odd that `NewSpotMarket` is separate - like it's only reasonable for historical reasons that it is separate. I'd prefer it to be merged in to `NewMarket`. But it's not a blocker.

Reply:
I'm just going by what is in the protos @edd, I think you're right though, I think it should be a configuration for a NewMarket request, perhaps raise it in a separate ticket as it has an impact on core